src/include/stir/ProjDataInfo.inl

```diff
@@ -185,7 +198,7 @@ ProjDataInfo::is_tof_data() const
     {
       if (num_tof_bins != 1)
         {
-          error("Non-TOF data with inconsistent Time-of-Flight bin number - Aborted operation.");
+          warning("Non-TOF data with inconsistent Time-of-Flight bin number - Note. As we are updating the scanner template style please make sure to have one TOF position (whole coincidence window for nonTOF scanners).");
```

Comment:
I don't think we need this. I never get the warning. It's a left over from when you were thinking the flag the special case in a different way.

Reply:
OK, commented out.